src/Bicep.RoslynAnalyzers/Bicep.RoslynAnalyzers.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- do not include the build output in the NuGet lib folder -->
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.0.1" PrivateAssets="all" />
```

Comment:
>4.0.1 [](http://example.com/codeflow?start=71&length=5)

These package versions are very strict. You have to target the version that is the lowest among all the tools we use (VS, VS code, dotnet, etc.)

Reply:
Should this comment go into the code?

Reply:
Yup!